apps/wa/wa-task-batch-reconfiguration-failure/ithc/00.yaml

```diff
@@ -5,6 +5,6 @@ metadata:
 spec:
   values:
     job:
-      schedule: "0,4,8,12,16,20,24,28,32,36,40,44,48,52,56 * * * *"
+      schedule: "1-60/2 * * * *"
```

Comment:
Aren't mins supposed to be from 0 to 59?

Reply:
Can review all the rest too I guess.

Reply:
Addressed the change